src/main/java/nl/hsac/fitnesse/fixture/slim/HttpTest.java

```diff
@@ -946,6 +946,27 @@ public boolean isFinished() {
         return repeatUntil(completion);
     }
 
+    public boolean repeatUntilResponseIsNot(final String expectedResponse) {
+        RepeatCompletion completion;
+        if (expectedResponse == null) {
+            completion = new RepeatLastCall() {
+                @Override
+                public boolean isFinished() {
+                    return response() == null;
```

Comment:
Shouldn't this be `!=`?

Reply:
Done by `repeatUntilNot`. I hope i understood you correctly.